frontend/package.json

```diff
@@ -1,15 +1,18 @@
 {
   "name": "bhopguessr",
   "private": true,
-  "version": "0.0.0",
+  "version": "0.0.1",
   "type": "module",
   "scripts": {
     "dev": "vite",
     "build": "tsc -b && vite build",
     "lint": "eslint . --ext ts,tsx --report-unused-disable-directives --max-warnings 0",
     "preview": "vite preview",
     "storybook": "storybook dev -p 6006",
-    "build-storybook": "storybook build"
+    "build-storybook": "storybook build",
+    "tests": "npx playwright test",
```

Comment:
We need to use pnpm instead.
For example to execute command via pnpm:
`pnpm exec playwright test`

Reply:
Done.